cmd/fossil/client/client.go

```diff
@@ -209,7 +209,7 @@ func readlinePrompt(c fossil.Client) {
 		}
 		msg, err := c.Send(replMsg)
 		if err != nil {
-			log.Error().Err(err).Msg("error sending message to server")
+			log.Fatal().Err(err).Msg("error sending message to server")
```

Comment:
To confirm, we want this to be fatal now that the client should retry sending and only error on an actual failure.

Reply:
Exactly. If we get an error back it means either something unexpected happened, or we exhausted our retries. In both cases, our client is probably stale so we should just exit.